content/rest-api/filter.md

```diff
@@ -1217,3 +1217,23 @@ Below you will find the operator allowed to filter on this property, as well as
 ```
 /api/rest/v1/categories?search={"code":[{"operator":"IN","value":["category_code1","category_code2"]}]}
 ```
+
+### By root category
+::: availability versions=Serenity editions=CE,EE
+
+You can filter the categories to get only root categories or non-root categories. Root categories can also be called Category trees. 
```

Comment:
```suggestion
You can filter the categories to get only root categories or non-root categories.
```

A root category is the base of the category tree. The category tree is a root category + some categories